### ISSUE TITLE
aiv: warn user if spi for same username is already open

### ISSUE THREAD

**Issue by NovemLinguae** [wikimedia-gadgets/twinkle#1496]
When sockmaster text box loses focus (person done typing), asynchronously grab the wikicode of that spi page. When sock text box loses focus, scan the stored wikitext for that sock name. Display a "possible duplicate" alert (red text by the text box) if found

Would help prevent this:
https://en.wikipedia.org/w/index.php?title=Wikipedia:Sockpuppet_investigations/JackNinte&diff=1061397890&oldid=1061384708&diffmode=source

**Comment by roysmith** [comment]
Yeah.  I literally just came here to ask for what I think is the same thing.  I'm looking at [Biancasuarez](https://en.wikipedia.org/w/index.php?title=Wikipedia:Sockpuppet_investigations/Biancasuarez&oldid=1065422186).  There's 3 open reports from the same reporter for Jan 11, Jan 13, and again Jan 13.  We see this often at SPI.  It would be really useful to the SPI team if the reporter was alerted to this.  You could even include a link which directly edits the "Suspected sockpuppets" section, i.e. "Please consider simply adding the new account(s) to the existing report.  You can do that by [clicking this link](https://en.wikipedia.org/w/index.php?title=Wikipedia:Sockpuppet_investigations/Biancasuarez&action=edit&section=2)".  If feasible, even more convenient would be for the script to just add the accounts itself.

You'll want some sanity checking.  It only makes sense to add additional accounts to cases in a few states.  Perhaps just "open" and "curequest".  Certainly not once a case is "in progress", "checked", or "closed".

**Comment by rsjaffe** [comment]
+1 on this issue. Just ran into it today. Annoying, and invisible to the user unless they open the spi page afterwards.

**Comment by NovemLinguae** [comment]
Requested again today. By Justlettersandnumbers at https://en.wikipedia.org/wiki/Wikipedia_talk:Twinkle#Duplicate_sockpuppet_reports

Diff of problem: https://en.wikipedia.org/wiki/Special:Diff/1189364242
